builder/builder.go

```diff
@@ -561,6 +561,19 @@ func (b *Builder) getMixBundlesListFull() ([]string, error) {
 	return bundleSet.Sort(), nil
 }
 
+func (b *Builder) getDirBundleList(dir string) ([]string, error) {
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to read bundles dir: %s", dir)
+	}
+
+	var bundles []string
```

Comment:
Since we know how long the list of bundles is going to be, we can avoid a certain amount of copying.

    var bundles = make([]string, len(files))
    for i := range files {
              bundles[i] = files[i].Name()
    }

Reply:
Yes, nice catch. I wrote the entire `mixer bundle list` functionality in about 10 minutes, so I am guessing there are other optimizations I didn't do as well.

Will fix in a future push.